app/src/main/java/com/orange/ods/app/ui/modules/emptystate/EmptyStateViewModel.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.app.ui.modules.emptystate
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+
+class EmptyStateViewModel: ViewModel() {
```

Comment:
```suggestion
class EmptyStateViewModel : ViewModel() {
```

FYI, it seems autoformat on save only works when I modify the file.